apidoc/Titanium/UI/View.yml

```diff
@@ -432,6 +432,19 @@ events:
             The time (in seconds) when the touch was used in correlation with the system start up.
             Note: This property is only available for iOS devices that support 3D-Touch and run 9.0 or later.
         type: Number
+        
+      - name: azimuthUnitVectorInViewX
+        summary: |
+            The x value of the unit vector that points in the direction of the azimuth of the stylus.
+            Note: This property is only available for iOS devices that supports apple pencil and are 9.1 or later.
```

Comment:
To: "Note: This property is only available for iOS devices that support the Apple Pencil and are 9.1 or later."

Please also change in the other new descriptions that are added.